src/Datadog.Trace/Tags.cs

```diff
@@ -214,6 +214,26 @@ public static class Tags
         /// </summary>
         public const string MessageSize = "message.size";
 
+        /// <summary>
+        /// The partition associated with a record
+        /// </summary>
+        public const string Partition = "partition";
```

Comment:
Can we prefix these with `kafka.`? Do you know what other tracers have done for tag names?

Reply:
I copied these from Java 🙂 https://github.com/DataDog/dd-trace-java/blob/e5f2514a90b04f4f7620d1497207b5a0224d0a3e/internal-api/src/main/java/datadog/trace/bootstrap/instrumentation/api/InstrumentationTags.java#L11-L15

I agree `kafka.` prefixes would be better, but I thought we'd probably need to stay consistent with other tracers?

Reply:
I would try to find an RFC or worst case scenario look at the source code for other tracers as well. Often Java implements features first (before there is a cross-language RFC), so copying their implementation is not always the best idea.

Reply:
Looks like ruby uses `kafka.partition` etc

https://github.com/DataDog/dd-trace-rb/blob/e63e65f19887d011cd957f7e58361f4af0df2050/lib/ddtrace/contrib/racecar/ext.rb#L19-L21

as does js:

https://github.com/DataDog/dd-trace-js/blob/7b27059fe3b9de6788c0517a2b499ff6b78da5ff/packages/datadog-plugin-kafkajs/src/index.js#L67-L69

So I'll add the `kafka.` prefixes 👍

Reply:
added `kafka.` in https://github.com/DataDog/dd-trace-dotnet/pull/1444/commits/0786ef38b54096f694424e973a1474dade8f5aac